docker-compose.yml

```diff
@@ -50,10 +50,26 @@ services:
       context: ./dff/utils/docker
       dockerfile: dockerfile_stats
     image: ghcr.io/deeppavlov/superset_df_dashboard:latest
```

Comment:
I can't seem to log in with the current build.
To reproduce:
1. Create a new github codespace at the current commit.
2. Remove this line to ensure local image build.
3. Try logging in with the `superset:superset` credentials or running `make test` or `dff.stats ...`. (`KeyError: 'access_token'`)

This is also an issue because GitHub Actions workflows are successful due to using `image` instead of `build` and thus using the master version of the dashboard.

Reply:
And could you also check if dashboard metadata persists in github codespaces after restarting containers?

Reply:
Updated; the config persists after restart